content/docs/tasks/traffic-management/secure-ingress/index.md

```diff
@@ -445,6 +445,80 @@ In this subsection, perform the same steps as in the [Generate client and server
         `"""`
     {{< /text >}}
 
+## Configure end-user authentication on ingress gateway
+
+The Istio ingress gateway supports end-user authentication by setting up a JWT
+authentication policy in the `istio-ingressgateway` file. The ingress gateway
+rejects the unauthenticated requests and the request can't access the services
+inside the mesh. This section shows how to use the authentication policy to
+setup the end-user authentication for the Istio ingress gateway.
+
+1. To verify the setup, run the following curl command and confirm a return
+   value of 200:
+
+    {{< text bash >}}
+    $ curl -I -HHost:httpbin.example.com http://$INGRESS_HOST:$INGRESS_PORT/status/200
+    {{< /text >}}
+
+1. Add the policy requiring the end-user JWT authentication for the
+   `istio-ingressgateway` pod.
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -f -
+    apiVersion: "authentication.istio.io/v1alpha1"
+    kind: "Policy"
+    metadata:
+      name: "ingressgateway"
+      namespace: istio-system
+    spec:
+      targets:
+      - name: istio-ingressgateway
+      origins:
+      - jwt:
+          issuer: "testing@secure.istio.io"
+          jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
+      principalBinding: USE_ORIGIN
+    EOF
+    {{< /text >}}
+
+1. Run the following curl command:
+
+    {{< text bash >}}
+    $ curl -I -HHost:httpbin.example.com http://$INGRESS_HOST:$INGRESS_PORT/status/200
+    {{< /text >}}
+
+1. Note the return value of 401. Istio returns this error code value because
+   the server expects a JWT but the ingress gateway did not provide one:
+
+    {{< text bash >}}
+    $ HTTP/1.1 401 Unauthorized
+    content-length: 29
+    content-type: text/plain
+    date: Mon, 13 Aug 2018 22:33:32 GMT
+    server: envoy
+    {{< /text >}}
+
+1.  Run the following curl command, attach the valid JWT:
```

Comment:
Fix grammar:
1. Run the following curl command with the valid JWT:

Reply:
thanks, updated.